recipes/iproute2/iproute2.inc

```diff
@@ -12,7 +12,7 @@ inherit autotools-autoreconf pkgconfig auto-package-utils
 SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/shemminger/iproute2.git;commit=${GIT_COMMIT}"
 S = "${SRCDIR}/iproute2"
 
-DEPENDS += "libdl libresolv libm libdb4 libnl-3 native:bison native:flex"
+DEPENDS += "libdl libresolv libm libnl-3 native:bison native:flex"
```

Comment:
Sounds nasty... but when no of the major distros depends on BerkDB I don't we should either.
I'll take a look in the configure script if it's easily turnoff